src/components/EventCard.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { ReactElement } from "react";
+import ".././styles.css";
+import PinSVG from ".././public/Pin.svg";
+import TextIconSVG from ".././public/TextIcon.svg";
+import MeatballMenuSVG from ".././public/MeatballMenu.svg";
+
+export interface EventCardProps {
+  startTime: Date;
+  endTime?: Date;
+  name: string;
+  location: string;
+  description: string;
+}
+
+/**
+ * Renders a single event in the feed
+ */
+const EventCard = ({
+  startTime,
+  endTime,
+  name,
+  location,
+  description,
+}: EventCardProps): ReactElement => {
+  let startTimeString: string = startTime.toLocaleString("en-US", {
+    hour: "numeric",
+    minute: "numeric",
+    hour12: true,
+  });
+  let endTimeString: string | undefined = endTime
+    ? endTime.toLocaleString("en-US", {
+        hour: "numeric",
+        minute: "numeric",
+        hour12: true,
+      })
+    : undefined;
+
+  return (
+    <>
+      <div className="pl-4 pt-4 pr-6 w-4/5">
+        <span className="font-sans">
+          {startTimeString + (endTime ? " to " + endTimeString : "")}
+        </span>
+        <div className="flex flex-row justify-between items-start mb-4">
+          <div className="not-italic font-bold text-2xl leading-8 font-sans break-words w-4/5">
+            {name}
+          </div>
+          <img
+            src={MeatballMenuSVG}
+            alt="Menu svg"
+            aria-label="Open Event Card details"
+          />
+        </div>
+        <div className="flex flex-row items-start mb-4">
+          <img src={PinSVG} alt="Pin svg" className="p-1 pt-0" />
+          <span className="text-body-mobile pl-2 pr-12 pt-0.5 font-montserrat break-words w-full">
+            {location}
+          </span>
+        </div>
+        <div className="flex flex-row items-start mb-6">
+          <img src={TextIconSVG} alt="TextIcon svg" className="p-1 pt-0" />
+          <p className="text-body-mobile pl-2 pr-12 font-montserrat break-words w-full">
+            {description}
+          </p>
+        </div>
+
+        <div className="flex flex-row flex-wrap">
+          <button className="button-base-white px-2 my-2 mr-5"> Unregister </button>
```

Comment:
omg it looks perfect now but i'm sorry super super small nit could we have these buttons positioned to the right side instead of the left similar to how it looks here in Josh's wireframe: 

<img width="175" alt="image" src="https://user-images.githubusercontent.com/40529597/199845199-343e5713-3cc6-4746-ad8b-1c39a79e236a.png">

If you'd rather keep it how it is now that works too! Stylistically I think I prefer it on the right side but this is a super small nit lol!

Reply:
<img width="358" alt="Screen Shot 2022-11-03 at 10 41 36 PM" src="https://user-images.githubusercontent.com/73146562/199873618-42751b37-ff25-456e-b2ba-1c0f9afa3e1c.png">

Ah I'm going off of the bottom events of the Figma, which is why I initially used `justify-between`. Come to think of it, I think it would make sense to pass in a `live` prop to the Event component, and if `live` then show the `Vote` button else show the other two buttons.

Reply:
Gotcha, having the `live` prop makes sense to me. Also, I'm also looking at the Figma desktop prototype and I think for larger screen sizes the buttons are actually aligned to the right, which may be what we want then for responsiveness: 

<img width="504" alt="image" src="https://user-images.githubusercontent.com/40529597/200040235-98c18061-c0e0-4b1f-95c4-7f8f614e0fa2.png">